code/modules/client/verbs/ooc.dm

```diff
@@ -468,3 +468,74 @@ ADMIN_VERB(reset_ooc_color, R_FUN, "Reset Player OOC Color", "Returns player OOC
 	set desc = "View the current map vote tally counts."
 	set category = "Server"
 	to_chat(mob, SSmap_vote.tally_printout)
+
+
+/client/verb/linkforumaccount()
+	set category = "OOC"
+	set name = "Link Forum Account"
+	set desc = "Validates your byond account to your forum account. Required to post on the forums."
+
+	var/forumlinklimit = 0
+	var/uri = CONFIG_GET(string/forum_link_uri)
+	if(!uri)
+		to_chat(src, span_warning("This feature is disabled."))
+		return
+
+	if (forumlinklimit > world.time + 10 SECONDS)
+		to_chat(src, span_userdanger("Please wait 10 seconds between forums link attempts."))
+		return
+
+	forumlinklimit = world.time
```

Comment:
Wait, forumlinklimit does absolutely nothing because the var resets to 0 before we check it (and it wouldn't keep state on line 488 regardless). Try using the COOLDOWN_DECLARE and COOLDOWN_* system on /client so you actually get a proper rate limiter... do we actually need one though?

Reply:
I think we can do without it for now, and if we need it again we can implement a proper cooldown.